Xamarin.Forms.Core/WeakEventListener.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.ComponentModel;
+
+namespace Xamarin.Forms
+{
+	[EditorBrowsable(EditorBrowsableState.Never)]
+    public sealed class WeakEventListener<TInstance, TSource, TEventArgs>
```

Comment:
Can we use WeakEventManager
https://github.com/xamarin/Xamarin.Forms/blob/c8cf3ba9edbc50bd2933515af39812949380d034/Xamarin.Forms.Core/WeakEventManager.cs#L10